projects.html

```diff
@@ -2,8 +2,273 @@
+    cursor: pointer;
+    font-size: 15px;
+    display: inline-block;
+    
+      text-align: center;
+    letter-spacing: 0.2125rem;
+    overflow: hidden;
+    background: rgba(255,255,255,.1);
+    position: relative;
+    text-align: center;
+    text-transform: uppercase;
+    border-radius: 30px;
+    z-index: 2;
+    box-shadow: 0 0 10px rgba(0,0,0.2);
+    
+    transition: 
+        background 5s cubic-bezier(0.19, 1, 0.22, 1),
+        border 1s cubic-bezier(0.19, 1, 0.22, 1),
+        color 0.6s cubic-bezier(0.19, 1, 0.22, 1);
+    user-select: none;
+  }
+  
+  .button a {
+    color: white;
+    font-family: "Varela Round";
+    text-decoration: none;
+  }
+  
+  .button:hover {
+    background-color: rgba(255, 255, 255, 0.05);
+    border-color: #fff;
+    box-shadow: 0 0 8px rgba(255, 245, 245, 0.8);
+    transition: background 0s;
+  }
+  .button:hover a {
+    color: #fff;
+  }
+   
+  .button1{
+	right: 27%;
+    top: -1080px;
+   }
+   .button2{
+	right: 10%;
+    top: -1050px;
+	  }
+   .button3{
+	left: 15%;
+    top: -460px;
+   }
+   .button4{
+	left: 50%;
+    top: -500px;
+   }
+   
+   .button5{
+	left: 15%;
+    top: -30px;
+   }
+   .t1{
+    top: -1480px;
+	left: 5%;
+	position: relative;
+	z-index: 3;
+	font-size:larger;
+	font-weight: bolder;
+
+   }
+   .t2{
+    top: -1520px;
+	left: 55%;
+	position: relative;
+	z-index: 3;
+	font-size:larger;
+	font-weight: bolder;
+
+   }
+   .t3{
+    top: -980px;
+	left: 30%;
+	position: relative;
+	z-index: 3;
+	font-size:larger;
+    font-weight: bolder;
+   }
+   .t4{
+    top: -1080px;
+	left: 55%;
+	position: relative;
+	z-index: 3;
+	font-size:larger;
+	font-weight: bolder;
+
+   }
+   .t5{
+    top: -480px;
+	left: 70%;
+	position: relative;
+	z-index: 3;
+	font-size:larger;
+	font-weight: bolder;
+
+   }
+
+   .glow {
+      -webkit-animation: glow 1s ease-in-out infinite alternate;
+     -moz-animation: glow 1s ease-in-out infinite alternate;
+     animation: glow 1s ease-in-out infinite alternate;
+}
+
+    @-webkit-keyframes glow {
+     from {
+    text-shadow: 0 0 10px whitesmoke, 0 0 20px white, 0 0 10px , 0 0 20px white, 0 0 50px white , 0 0 30px white , 0 0 30px ;
+     }
+  
+     to {
+    text-shadow: 0 0 20px #fff, 0 0 10px orangered, 0 0 20px orangered , 0 0 20px orangered, 0 0 10px orangered, 0 0 10px orangered , 0 0 10px orangered ;
+     }
+     }
+  
+   body{
+    overflow-x: hidden;
+    background-color: black;
+	background-image: url('img/bg.jpg');
+	background-repeat: no-repeat;
+    background-attachment: fixed;
+   }
+
+		</style>
 </head>
 <body>
+	<!-- IMAGES -->
+
+	<img src="img/p1.png" class="img1">
+	<img src="img/p2.png" class="img2">
+	<img src="img/p3.jpg" class="img3">
+	<img src="img/p4.jpg" class="img4">
+	<img src="img/p5.jpg" class="img5">
+
+
+	<!-- BUTTONS -->
+
+	<div class="button button1" >
+		<a href="https://github.com/DSC-RSU/DSC-RSU-Website">
+		<font color="black">Link To Project</font>
+		</a>
+	</div>
+
```

Comment:
i thought by adding it, it will be more easy to read the code
and it'll look neat

Reply:
That's true but actually we dont break the rules of indentations, so it is always recommended to keep one tab and no new lines after any tag. Also tag's indentation should only be changed if it is one inside other, otherwise follow the same indentation.

Reply:
okk, I'll keep in mind